src/frontend/shortlist/src/api/shortlist.js

```diff
@@ -33,14 +33,18 @@ export default class ShortlistApi {
   addStudent(payload, successCb, failCb) {
     return new temporaryAddStudent(payload, successCb, failCb);
   }
+  authLogin(payload, successCb, failCb) {
+    return new tempAuthLogin(payload, successCb, failCb);
+  }
 }
 
 import axios from "axios";
 import md5 from "md5";
 export class temporarySignup {
   constructor(payload, successCb, failCb) {
     this.email = payload.email;
-    this.username = md5(payload.email).substring(0, 10);
+    // this.username = md5(payload.email).substring(0, 10);
+    this.username = payload.firstName + payload.lastName;
```

Comment:
This is more of a question for me: do we now use the combination of firstname and lastname as the primary key?

Reply:
There is a user_id which is the primary key, right now the username is only used as a title for sending email

Reply:
I see, I thought that was commented out